Wrong defaults in business-api-ecosystem values

The business-api-ecosystem chart throws the following warnings upon installation due to incorrect default values for [`bizEcosystemLogicProxy.extraVolumeMounts`](https://github.com/FIWARE/helm-charts/blob/4983fd741b9c0add638f7a4292bbb9c1f2ac90da/charts/business-api-ecosystem/values.yaml#L826) and [`bizEcosystemLogicProxy.command`](https://github.com/FIWARE/helm-charts/blob/4983fd741b9c0add638f7a4292bbb9c1f2ac90da/charts/business-api-ecosystem/values.yaml#L828C3-L828C10):

```shell
coalesce.go:298: warning: cannot overwrite table with non table for datasphere.marketplace.bizEcosystemLogicProxy.extraVolumeMounts (map[])
coalesce.go:298: warning: cannot overwrite table with non table for datasphere.marketplace.bizEcosystemLogicProxy.command (map[])
```

Default values expected by the themplates should be empty arrays instead of empty objects.

Tested with the following versions:

- Helm v3.18.3
- BAE chart v0.11.19

--- a/charts/business-api-ecosystem/Chart.yaml
+++ b/charts/business-api-ecosystem/Chart.yaml
@@ -2,7 +2,7 @@ apiVersion: v2
 name: business-api-ecosystem
 description: A Helm chart for running the FIWARE business API ecosystem (FIWARE Marketplace) on Kubernetes
 icon: https://fiware.github.io/catalogue/img/fiware.png
-version: 0.11.19
+version: 0.11.20
 appVersion: 9.0.1
 home: https://business-api-ecosystem.readthedocs.io/en/latest/
 keywords:

--- a/charts/business-api-ecosystem/README.md
+++ b/charts/business-api-ecosystem/README.md
@@ -1,6 +1,6 @@
 # business-api-ecosystem
 
-![Version: 0.11.19](https://img.shields.io/badge/Version-0.11.19-informational?style=flat-square) ![AppVersion: 9.0.1](https://img.shields.io/badge/AppVersion-9.0.1-informational?style=flat-square)
+![Version: 0.11.20](https://img.shields.io/badge/Version-0.11.20-informational?style=flat-square) ![AppVersion: 9.0.1](https://img.shields.io/badge/AppVersion-9.0.1-informational?style=flat-square)
 
 A Helm chart for running the FIWARE business API ecosystem (FIWARE Marketplace) on Kubernetes
 
@@ -128,7 +128,7 @@ A Helm chart for running the FIWARE business API ecosystem (FIWARE Marketplace) 
 | bizEcosystemLogicProxy.allowLocalEORI | bool | `false` | Allow to use organisations from local IDP as participants when creating or acquiring offerings |
 | bizEcosystemLogicProxy.basePath | string | `"/business-ecosystem-logic-proxy"` | Base app path of logic proxy (for versions < 8.1.0: /business-ecosystem-logic-proxy, for versions >= 8.1.0: /opt/business-ecosystem-logic-proxy) |
 | bizEcosystemLogicProxy.collectStaticCommand | string | `"True"` | Execute the collect static command on startup |
-| bizEcosystemLogicProxy.command | object | `{}` | in case the startup command should be alterd |
+| bizEcosystemLogicProxy.command | list | `[]` | in case the startup command should be alterd |
 | bizEcosystemLogicProxy.db.database | string | `"belp_db"` | Database name for connecting the database |
 | bizEcosystemLogicProxy.db.host | string | `"mongo"` | host of the database to be used |
 | bizEcosystemLogicProxy.db.password | string | `"pass"` | password for connecting the database |
@@ -141,7 +141,7 @@ A Helm chart for running the FIWARE business API ecosystem (FIWARE Marketplace) 
 | bizEcosystemLogicProxy.enabled | bool | `true` |  |
 | bizEcosystemLogicProxy.externalIdp.enabled | bool | `false` | Enable usage of external IDPs |
 | bizEcosystemLogicProxy.externalIdp.showLocalLogin | bool | `false` | Show login button for local IDP on login modal dialog with list of external IDPs |
-| bizEcosystemLogicProxy.extraVolumeMounts | object | `{}` |  |
+| bizEcosystemLogicProxy.extraVolumeMounts | list | `[]` |  |
 | bizEcosystemLogicProxy.fullnameOverride | string | `""` |  |
 | bizEcosystemLogicProxy.ingress.annotations | object | `{}` | annotations to be added to the ingress |
 | bizEcosystemLogicProxy.ingress.enabled | bool | `false` | should there be an ingress to connect the logic proxy with the public internet |

--- a/charts/business-api-ecosystem/values.yaml
+++ b/charts/business-api-ecosystem/values.yaml
@@ -823,6 +823,6 @@ bizEcosystemLogicProxy:
 
   # -- use initcontainers to wait for the apis to be deployed
   initContainers: true
-  extraVolumeMounts: {}
+  extraVolumeMounts: []
   # -- in case the startup command should be alterd
-  command: {}
+  command: []

